modules/getting-started-cli-deploying-first-image.adoc

```diff
@@ -1,21 +1,23 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-cli.adoc
+// * tutorials/dev-app-cli.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-cli-deploying-first-image_{context}"]
-= Deploying your first image
+= Deploying the front-end application
 
-The simplest way to deploy an application in {product-title} is to run an existing container image. The following procedure deploys a front-end component of an application called `national-parks-app`. The web application displays an interactive map. The map displays the location of major national parks across the world.
+The simplest way to deploy an application in {product-title} is to run a provided container image.
+
+The following procedure deploys `parksmap, which is the front-end component of the `national-parks-app` application. The web application displays an interactive map of the locations of major national parks across the world.
```

Comment:
There's a typo here rendering a faulty visualization of the text in the web version. There's a lacking backtick. Please, fix it as following:

The following procedure deploys `parksmap`, which is the front-end component of the `national-parks-app` application. The web application displays an interactive map of the locations of major national parks across the world.